### DISCUSSION TITLE
What is Tick Tick?

### DISCUSSION THREAD

**Discussion by githubUser01946** [ReVanced/revanced-discussions#821] [Question] [answered]
I saw there was something called TickTick and wondered what it was, TikTok was also listed so Im not old and confused :)

**Comment by oSumAtrIX** [comment, marked as answer]
This yields results:

<img width="313" alt="image" src="https://user-images.githubusercontent.com/13122796/207562304-512c6fe1-512e-4557-8d4b-2f13f3324e63.png">

**Reply by githubUser01946** [reply]
Ohh so a playstore alternative, ok cool.

**Reply by ghost** [reply]
> Ohh so a playstore alternative, ok cool.

Tick Tick is a Task management app.
